docs/reference/release-notes/highlights.asciidoc

```diff
@@ -106,6 +106,22 @@ search features like queries and aggregations.
 
 {es-pull}88694[#88694]
 
+[discrete]
+[[synthetic_source_technical_preview]]
+=== Synthetic `_source`
+When Elasticsearch ingests documents, it creates several data structures that
+enable querying and aggregating the data. One of those data structures is called
+`_source`.  `_source` holds the fields of every document. It uses a lot of space
+and is used for specific needs, like reindexing and presenting the document in
+Kibana Discover. 
+
+This release introduces a new feature in technical preview called synthetic
+`_source`. For specific data types, synthetic `_source` can recreate the
+`_source` from doc values when needed. This option significantly reduces the
+index size.
```

Comment:
I'd love to hear more about the benefits of this feature, such as when and where folks would typically use it. It reduces the index size, but perhaps we can provide information about when that's necessary? Something like:

>Synthetic `_source` significantly reduces the index size by recreating the `_source` from doc values, and is useful for when you don't need to store everything in `_source`. While doc values don't preserve the original structure of the `_source`, they're enough to recreate the `_source` and still allow for capabilities like reindexing.

I'm taking some liberties based on a quick review of the related PR, so don't accept that suggestion verbatim 😆

Reply:
Thanks @lockewritesdocs . I've pushed a change that incorporates your feedback.